taxi/tests/test_admin.py

```diff
@@ -0,0 +1,49 @@
+from django.urls import reverse
+
+from django.contrib.auth import get_user_model
+from django.test import TestCase, Client
+
+
+class AdminSiteTests(TestCase):
+    def setUp(self):
+        self.client = Client()
+        self.admin_user = get_user_model().objects.create_superuser(
+            username="admin",
+            password="testadmin"
+        )
+        self.client.force_login(self.admin_user)
+        self.driver = get_user_model().objects.create_user(
+            username="driver",
+            password="testdriver",
+            license_number="TST12345"
```

Comment:
The 'license_number' field is being used in the test, but it is not a default field for Django's User model. Ensure that the User model has been extended to include this field, or this test will fail.